six/modules/c++/six.sicd/include/six/sicd/Utilities.h

```diff
@@ -40,11 +40,35 @@ namespace sicd
 class Utilities
 {
 public:
+    // The following three methods return raw pointers because the SWIG wrapper
+    // complains about auto_ptrs
+
+    /*!
+     * Build SceneGeometry from ComplexData members
+     * \param data ComplexData from which to construct Geometry
+     * \return SceneGeometry from ComplexData
+     */
     static scene::SceneGeometry* getSceneGeometry(const ComplexData* data);
 
+    /*!
+     * Build ProjectionModel from ComplexData members
+     * \param data ComplexData from which to construct ProjectionModel
+     * \param geom SceneGeometry for the model
+     * \return ProjectionModel from complexData
+     */
     static scene::ProjectionModel* getProjectionModel(const ComplexData* data,
             const scene::SceneGeometry* geom);
 
+    /*!
+     * Build ProjectionPolynomialFitter from complexData and
+     * given GriddedDisplayType.
+     * This always uses a PlanarGridECEFTransform
+     * \param complexData ComplexData from which to construct fitter
+     * \return ProjectionPolynomialFitter from ComplexData
+     */
+    static scene::ProjectionPolynomialFitter*
+    getPolynomialFitter(const ComplexData& complexData);
```

Comment:
I think the two old methods use raw pointers just because we didn't used to be pedantic about smart pointers.  Can Swig really not work with this?  I thought we've gotten this to work before.  In `mem.i` we use `%include "std_auto_ptr.i"` which I thought enabled Swig to do some magic.